src/agent/src/storage/fs_handler.rs

```diff
@@ -49,6 +51,18 @@ impl StorageHandler for OverlayfsHandler {
             storage
                 .options
                 .push(format!("workdir={}", work.to_string_lossy()));
+        } else if storage
```

Comment:
How about using `storage.driver_options` instead of `storage.options` to passing the option?
```
storage.driver_options.push("io.katacontainers.volume.overlayfs.create_directory=/xxx/cid/fs");
storage.driver_options.push("io.katacontainers.volume.overlayfs.create_directory=/xxx/cid/work");
```

Reply:
Done.